developer_manual/general/codingguidelines.rst

```diff
@@ -39,7 +39,7 @@ The most important labels and their meaning:
  * #5 - To Review - ready for review
  * #6 - Reviewing - review in progress
  * #7 - To Release - reviewed PR that awaits unfreeze of a branch to get merged
-* App tags: #app:files #app:user_ldap #app:files_encryption and so on. These tags indicate the app that is impacted by the issue or which the PR is related to
+* App tags: #app:files #app:user_ldap #app:files_version and so on. These tags indicate the app that is impacted by the issue or which the PR is related to
```

Comment:
I changed this sample, to avoid confusions because of app vs. feature on the encryption tags.